### PR TITLE
mergerfs: 2.32.4 -> 2.32.6

### PR DESCRIPTION
**Pull request by eadwu**
###### Motivation for this change
Have been running `mergerfs` for the past few days. Anything from `mergerfs-tools` though not really.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 131636` at 4e46968d run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built successfully:</summary>
  <ul>
    <li>mergerfs</li>
    <li>mergerfs-tools</li>
  </ul>
</details>

File: pkgs/tools/filesystems/mergerfs/tools.nix [hunk 1]
```diff
@@ -1,30 +1,36 @@
 { lib, stdenv, fetchFromGitHub, coreutils, makeWrapper
-, rsync, python3, pythonPackages }:
+, rsync, python3Packages }:
```
Comment (SuperSandro2000) [inline pkgs/tools/filesystems/mergerfs/tools.nix:2]:
```suggestion
, rsync, python3 }:
```

File: pkgs/tools/filesystems/mergerfs/tools.nix [hunk 2]
```diff
@@ -1,30 +1,36 @@
...
     repo = pname;
-    rev = "6e41fc5848c7cc4408caea86f3991c8cc2ac85a1";
-    sha256 = "0izswg6bya13scvb37l3gkl7mvi8q7l11p4hp4phdlcwh9jvdzcj";
+    rev = "3b6fe008517aeda715c306eaf4914f6f537da88d";
+    sha256 = "sha256-TQGlvFBYoVO8Qa8rAfIbTgTIsW1I3MfYKT5uwkz175Y=";
   };
 
   nativeBuildInputs = [ makeWrapper ];
-  buildInputs = [ python3 ];
+  buildInputs = [ python3Packages.python ];
```
Comment (SuperSandro2000) [inline pkgs/tools/filesystems/mergerfs/tools.nix:16]:
```suggestion
  buildInputs = [ python3 ];
```

File: pkgs/tools/filesystems/mergerfs/tools.nix [hunk 3]
```diff
@@ -1,30 +1,36 @@
...
+    let
+      binPath = lib.makeBinPath [
+        rsync
+        python3Packages.xattr
+      ];
+    in ''
+      for program in $out/bin/mergerfs.*; do
+        wrapProgram "$program" --prefix PATH: ${binPath}
+      done
+    '';
```
Comment (SuperSandro2000) [inline pkgs/tools/filesystems/mergerfs/tools.nix:33]:
```suggestion
  postInstall = ''
      for program in $out/bin/mergerfs.*; do
        wrapProgram "$program" --prefix PATH: ${lib.makeBinPath [ rsync python3.pkgs.xattr ]}
      done
    '';
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: pkgs/tools/filesystems/mergerfs/tools.nix (hunk 3 above)
Comment (makefu) [reply pkgs/tools/filesystems/mergerfs/tools.nix:33]: tbh, both solutions are absolutely fine. thanks for cleaning this up

**Review by makefu — APPROVED** [review]
With or without proposed changes by @SuperSandro2000 this PR looks good to me!

File: pkgs/tools/filesystems/mergerfs/tools.nix [hunk 4]
```diff
@@ -20,11 +20,17 @@ stdenv.mkDerivation rec {
...
+  postInstall =
+    let
+      binPath = lib.makeBinPath [
+        rsync
+        python3.pkgs.xattr
+      ];
+    in ''
+      for program in $out/bin/mergerfs.*; do
+        wrapProgram "$program" --prefix PATH: ${binPath}
+      done
```
Comment (SuperSandro2000) [inline pkgs/tools/filesystems/mergerfs/tools.nix:32]:
```suggestion
  postInstall = ''
      for program in $out/bin/mergerfs.*; do
        wrapProgram "$program" --prefix PATH: ${lib.makeBinPath [ rsync python3.pkgs.xattr ]}
      done
```

File: pkgs/tools/filesystems/mergerfs/tools.nix [hunk 5]
```diff
@@ -20,11 +20,17 @@ stdenv.mkDerivation rec {
...
+    let
+      binPath = lib.makeBinPath [
+        rsync
+        python3.pkgs.xattr
+      ];
+    in ''
+      for program in $out/bin/mergerfs.*; do
+        wrapProgram "$program" --prefix PATH : ${binPath}
+      done
+    '';
```
Comment (SuperSandro2000) [inline pkgs/tools/filesystems/mergerfs/tools.nix:33]:
```suggestion
  postInstall =
    let
      binPath = ;
    in ''
      for program in $out/bin/mergerfs.*; do
        wrapProgram "$program" --prefix PATH : ${lib.makeBinPath [ rsync python3.pkgs.xattr ]}
      done
    '';
```

**Pull request closed without merging**